ciao-launcher/qemu.go

```diff
@@ -108,7 +108,7 @@ func (q *qemuV) ensureBackingImage() error {
 	return nil
 }
 
-func (q *qemuV) createImage(bridge string, userData, metaData []byte) error {
+func (q *qemuV) createImage(bridge, gatewayIP string, userData, metaData []byte) error {
```

Comment:
@markdryan Why are we doing this for VMs. Today we push that automatically via DHCP. Also for a CNCI VM, we do not want this pushed in.

Reply:
It should not cause issues, but I am worried about potential conficts between this and what we serve via DHCP

Reply:
We're not.  The value is ignored in this file.  It has to be passed however, otherwise qemuV will not implement the virtualizer interface.

Reply:
Maybe the virtualizer interface needs to be tidied up a bit, but this a current restriction.  All virtualizers get passed all information, whether they need it to create their specific instance types or not.